README.md

```diff
@@ -18,33 +36,29 @@ This new Custom role along with the built-in "Reader" role is applied to the sub
 3. Run the following
 ```bash
 curl -s https://raw.githubusercontent.com/cloudconformity/temp-azure-custom-policy/master/apply-roles | bash /dev/stdin \
-    --application-id <App registration client id> \
-    --subscription-id <Subscription id>
+    --application-id fj63c3b9-46e3-41ca-bf79-344a076fa9b2
 ```
-`<App client id>` is the Application (client) ID of the App registration that Cloud One Conformity will be given access to
-`<Subscription id>` is the id of the Subscription to apply the role and permissions to.
 
 ### On local terminal
 
 #### Pre-requisites
-1. Bash version >= 4.
-2. [jq](https://stedolan.github.io/jq/).
+1. Bash version >= 4
+2. [jq](https://stedolan.github.io/jq/)
 
-_Note: Both of these are available by default in the Azure Cloud Shell_.
+_Note: Both of these are available by default in the Azure Cloud Shell._
 
 #### Running the script
 1. Install Azure CLI ([Installation instructions](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest)).
-1. Download bash script to local machine from Github repo.
 2. Log into your Azure account using `az login`.
+3. Download bash script to local machine from Github repo.
+4. Make the following modifications to the bash script
+   1. Change the value of the `GITHUB_URI` variable to `"."`.
+   2. Rename all reference to the `--template-uri` parameter to `--template-file`.
 3. Locate the id of the App Registration that the custom role is to be applied to.
 4. In the same directory as the bash script run.
 ```bash
```

Comment:
❓ should the above be updated to step 5 and 6? Due to newly added steps (steps 3 and 4)?

Reply:
Markdown doesn't care about the order of the numbers just that they are numbers. When it is rendered it numbers them in the order they appear in the list and ignores these numbers